Header header
int16 heading    # heading in radians
int16[2] velocity  # velocity in rpm/s